Content/Maatregelen/M01/Maatregel.md

```diff
@@ -131,6 +139,7 @@ De onderstaande tabel bevat de hierboven genoemde producten. Het ✔ geeft aan i
 | Broncode, inclusief de benodigdheden voor het bouwen van de software                                    |          | ✔              |
 | Regressietests, inclusief de benodigdheden voor het uitvoeren van de tests                              |          | ✔              |
 | Deploymentdocumentatie                                                                                  |          | ✔              |
+| Software Bill of Materials                                                                              |          | ✔              |
```

Comment:
Waarom met hoofdletters? Het is geen naam toch?

Reply:
Macht der gewoonte. Aangepast.